packages/test/src/test-sinks.ts

```diff
@@ -15,6 +21,21 @@ class DependencyError extends Error {
   }
 }
 
+function asDefaultLoggerSink(
+  fn: (info: WorkflowInfo, message: string, attrs?: Record<string, unknown>) => Promise<void>,
+  opts?: Omit<InjectedSinkFunction<any>, 'fn'>
+): InjectedSinks<DefaultLoggerSinks> {
+  return {
+    defaultWorkerLogger: {
+      trace: { fn, ...(opts ?? {}) },
+      debug: { fn, ...(opts ?? {}) },
+      info: { fn, ...(opts ?? {}) },
+      warn: { fn, ...(opts ?? {}) },
+      error: { fn, ...(opts ?? {}) },
```

Comment:
Just `...opts` works, you can splat an undefined | null variable

Reply:
No, really?!? I always thought the spread operator was not valid on null/undefined values. Turns out that's only true for array spread, not for object spread. TIL!